build/media-suite_compile.sh

```diff
@@ -2424,6 +2426,75 @@ if [[ $ffmpeg != no ]]; then
     fi
 fi
 
+_check=(libde265.a)
+[[ $standalone = y ]] && _check+=(bin-video/dec265.exe)
+if [[ $libheif = y ]] &&
+    do_vcs "$SOURCE_REPO_LIBDE265"; then
+    do_uninstall "${_check[@]}"
+    extracommands=()
+    [[ $standalone = n ]] && extracommands+=(-DENABLE_{DE,EN}CODER=OFF)
+    do_cmakeinstall video "${extracommands[@]}"
+    do_checkIfExist
+fi
+
+_check=(bin-video/heif-{dec,enc,info,thumbnailer}.exe)
+if [[ $libheif = y ]] &&
+    do_vcs "$SOURCE_REPO_LIBHEIF"; then
+    do_uninstall "${_check[@]}"
+
+    sed -i 's/find_package(vvdec 2.3.0)/find_package(vvdec 3.0.0)/' CMakeLists.txt
+    sed -i 's/find_package(Doxygen)/#/' CMakeLists.txt # no configurable option?
+    sed -i 's/find_package(Brotli)/#/' CMakeLists.txt # linking difficulties
+    sed -i 's/find_package(TIFF)/#/' heifio/CMakeLists.txt # configure & linking difficulties
```

Comment:
I would prefer to fix these upstream if possible. If not, or if upstream rejects changes to fix these, then I am fine with adding them here.

Reply:
Man, I don't think they are going to be fixed anytime soon, maybe we can merge it for now, or transfer them into patches?

Reply:
I would prefer transferring to patches since it makes it easier for me to know if it breaks or is no longer needed.

Reply:
Can you push to the branch and patches repo when you decide to merge this PR? Or shall I do it and also open a PR in the patches repo?

Reply:
I would prefer that you do it so the authorship of the changes is preserved